src/test/resources/examples/responsesSchema/controllers/spring/Controllers.kt

```diff
@@ -0,0 +1,23 @@
+package examples.responsesSchema.controllers
+
+import examples.responsesSchema.models.SucccessResponse
+import org.springframework.http.ResponseEntity
+import org.springframework.stereotype.Controller
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestMethod
+
+@Controller
+@Validated
+@RequestMapping("")
+interface FooController {
+    /**
+     *
+     */
+    @RequestMapping(
+        value = ["/foo"],
+        produces = ["application/json"],
+        method = [RequestMethod.POST]
+    )
+    fun post(): ResponseEntity<SucccessResponse>
```

Comment:
**nitpick (blocking):** Succcccccccess? 😆

Reply:
Please rename in all applicable places

Reply:
To be fair, my change is perfectly fine, as the typo already exists in the yml spec :)

I'll fix it though.

Reply:
My apologies then. :)

Reply:
No need to apologize. Even though this wasn't introduced by my change, it's still a mistake that should be fixed, so great catch, Eagle-Eye.